src/config.ts

```diff
@@ -114,6 +124,7 @@ const sourceFees: { [key in ERC20BridgeSource]: BigNumber } = {
     [ERC20BridgeSource.CurveUsdcDaiUsdtTusd]: new BigNumber(8e5),
     [ERC20BridgeSource.CurveUsdcDaiUsdtBusd]: new BigNumber(8e5),
     [ERC20BridgeSource.Kyber]: new BigNumber(8e5),
+    [ERC20BridgeSource.LiquidityProvider]: new BigNumber(8e5),
```

Comment:
@fragosti I'm really unsure what I should be putting here as a source fee, who should I be asking?

Reply:
How much gas does a LiquidityProvider trade cost? Then add an additional 150e3

Reply:
@hysz much gas does a LiquidityProvider trade cost?

Reply:
The total gas cost for a LiquidityProvider trade routed through the 0x Exchange contract should cost ~250k gas. Note that as new liquidity providers come online we may have to adjust this value.